src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -279,14 +275,18 @@ private void setupKeyboardShortcuts() {
                 }
             }
             if (MANAGE_ASSIGNEES.match(event) && ui.getBrowserComponent().isCurrentUrlIssue()) {
-                ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                if (ui.getBrowserComponent().isCurrentUrlPR()) {
+                    ui.getBrowserComponent().openPrTab(getCurrentlySelectedIssue(), DISCUSSION_TAB);
+                }
                 ui.getBrowserComponent().manageAssignees(event.getCode().toString());
             }
             if (SHOW_MILESTONES.match(event)) {
                 if (KeyPress.isValidKeyCombination(GOTO_MODIFIER.getCode(), event.getCode())) {
                     ui.getBrowserComponent().showMilestones();
                 } else if (ui.getBrowserComponent().isCurrentUrlIssue()) {
-                    ui.getBrowserComponent().switchToTab(DISCUSSION_TAB);
+                    if (ui.getBrowserComponent().isCurrentUrlPR()) {
+                        ui.getBrowserComponent().openPrTab(getCurrentlySelectedIssue(), DISCUSSION_TAB);
```

Comment:
I think it's wrong to use `isCurrentUrlPr` here. You also need to create something new or update `isPullRequestLoaded` accordingly on `GitHubURL`.

So the intention here of switching tab is that if `isCurrentUrlIssue`, it means that we can be in an issue page or PR page in any of the tab. The call `switchToTab(DISCUSSION_TAB)` will not do anything on issue page anyway, so we can just call it here, and if we are on PR page, it ensures that we go to `DISCUSSION_TAB`.

The thing is, `isCurrentUrlPR` checks if we are in PR page, of any tab. Hence the call here is redundant and unnecessary. You should have a new regex (in `GitHubUrl`) that checks if we are in PR page but not discussion tab (by checking the ending of the URL, `files` and `commits`). If we are not, then open the discussion tab.